README.md

```diff
@@ -65,40 +56,17 @@ There are three ways to get changes made to the documentation. Two are 'do it yo
 
 ## Preview the documentation
 
-### Preview Sphinx/reST documentation with Docker
-
-To build the documentation (convert `.rst` files into HTML), you must [install Sphinx and extensions](#install-sphinx-and-extensions).
+### With Docker
 
-If you have [Docker installed](https://docs.docker.com/get-docker/), a more convenient way is to use our Docker image as follows:
+1. Install [Docker](https://docs.docker.com/get-docker/).
 
-1. Clone this repository.
+2. Clone this repository.
 
-2. `cd pmm-doc`
+3. `cd pmm-doc`
 
-3. `docker run --rm -v $(pwd):/docs perconalab/percona-doc-sphinx make clean html`
+4. `docker run --rm -v $(pwd):/docs perconalab/pmm-doc-md`
 
-4. Open `build/html/index.html` in a browser to view the first page of documentation.
-
-> **Tip**
->
-> Documentation built this way has no styling because it is intended for hosting on percona.com.
-> You can build a themed version for local viewing by changing the command in step 3 to:
->
-> `docker run --rm -v $(pwd):/docs perconalab/percona-doc-sphinx make clean thtml`
-
-### Preview MkDocs/md documentation with Docker
-
-To build the documentation (convert `.md` files into HTML), you must [install MkDocs and extensions](#install-mkdocs-and-extensions).
-
-If you have [Docker installed](https://docs.docker.com/get-docker/), a more convenient way is to use our Docker image as follows:
-
-1. Clone this repository.
-
-2. `cd pmm-doc`
-
-3. `docker run --rm -v $(pwd):/docs perconalab/pmm-doc-md`
-
-4. Open `site/index.html` in a browser to view the first page of documentation.
+5. Open `site/index.html` in a browser to view the first page of documentation.
```

Comment:
For clarity sake, I suggest the provide an url
```suggestion
5. Open `https://localhost/site/index.html` in a browser to view the first page of documentation.
```

Reply:
It's a static file on the local filesystem. It's not being served by a web server (as it is when running `mkdocs serve`).

Reply:
Ok, makes sense.